CHANGELOG.md

```diff
@@ -19,3 +19,4 @@
 * [ENHANCEMENT] Add spanlogger package. #42
 * [BUGFIX] spanlogger: Support multiple tenant IDs. #59
```

Comment:
[nit, unrelated] Do you mind moving this after the groups of `[ENHANCEMENT]` entries, please?